widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/TimeUtil.java

```diff
@@ -586,23 +621,11 @@ public static int getDayVariance( int selectedHour, int selectedMinute, String t
    * @param isTimezoneId True if the format is timezone id, false if the format is dateTime.
    * @return The target's timezone offset
    */
-  private static int getTargetOffset( String targetTimezoneInfo, boolean isTimezoneId ) {
-    String targetTzOffset;
-    int targetOffset;
-
+  private static double getTargetOffset( String targetTimezoneInfo, boolean isTimezoneId ) {
     if ( isTimezoneId ) {
```

Comment:
This could be refactored to a ternary operation :)

Reply:
Yes it could! and now it is :) 👍